### PR TITLE
CM-129: Using Analyzers in ASP.NET Core Web API

### PR DESCRIPTION
**Pull request by muhammedsalimp**
CM-129: Using Analyzers in ASP.NET Core Web API

### CODE REVIEW

File: aspnetcore-webapi/APIAnalyzers/WebAPIAnalyzers/CustomConventions.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesDefaultResponseType]
+    [ApiConventionNameMatch(ApiConventionNameMatchBehavior.Prefix)]
+    public static void Get(
+        [ApiConventionNameMatch(ApiConventionNameMatchBehavior.Suffix)]
+        int id)
+    { }
+}
+}
```
Comment (mahrncic) [inline aspnetcore-webapi/APIAnalyzers/WebAPIAnalyzers/CustomConventions.cs:18]: Can you please fix the indentation in this class? You can see that our `CustomConventions` class is not indented properly when comparing it to the namespace.

**Review by mahrncic — CHANGES_REQUESTED** [review]

**Review by mahrncic — APPROVED** [review]
Looks good.

File: aspnetcore-webapi/APIAnalyzers/WebAPIAnalyzers/FakeRepository.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        public List<Employee>? Employees { get; set; }
+
+        public IEnumerable<Employee> GetEmployees()
+        {
+            Employees = new()
+            {
+                new Employee()
+                {
+                    Id = 1,
+                    Name = "John Doe"                },
```
Comment (mahrncic) [inline aspnetcore-webapi/APIAnalyzers/WebAPIAnalyzers/FakeRepository.cs:20]: Just a small note, can you please fix the indentation/formatting of this bracket?

**Review by mahrncic — CHANGES_REQUESTED** [review]
It looks good, just one small change regarding formatting :)

File: aspnetcore-webapi/APIAnalyzers/WebAPIAnalyzers/FakeRepository.cs (hunk 2 above)
Comment (muhammedsalimp) [reply aspnetcore-webapi/APIAnalyzers/WebAPIAnalyzers/FakeRepository.cs]: corrected :)

**Review by mahrncic — APPROVED** [review]

**Pull request merged**